packages/dataviews/README.md

```diff
@@ -1046,11 +1086,91 @@ Example:
 // Configure a filter as multi-selection with all the options.
 {
 	filterBy: {
-		operators: [ `isAny`, `isNone`, `isAll`, `isNotAll` ]
+		operators: [ `isAny`, `isNone`, `isAll`, `isNotAll` ];
 	}
 }
 ```
 
+## Form Field API
```

Comment:
Added documentation around the `FormField` type, that is supported by `form.fields`